CHANGES.rst

```diff
@@ -1,6 +1,6 @@
 Changes
 =======
 
-Version 0.1 (released FIXME):
+Version 0.1 (released 2015-02-25):
 
 - Initial public release on PyPI.
```

Comment:
"Initial public release." only, so that the text would match what we have in `RELEASE-NOTES.rst`.